### PR TITLE
Management

### LINKED ISSUE #4: Tickets
HIGH LEVEL.
- [x] #5
- [x] #6
- [x] #7
- [x] #8

### LINKED ISSUE #5: Page to see all tickets
Should be a table showing all the current tickets

### LINKED ISSUE #6: Page to add a new ticket
This page will show the text boxes or dropdowns to create a new ticket

### LINKED ISSUE #7: Page to edit a ticket
Page design can be a copy of #6 but it must edit a ticket

MUST ALSO INCLUDE AN ITERATION COMMENT

### LINKED ISSUE #8: Button to disable a ticket
This should probably be on the #7 edit page that will disable a ticket

### LINKED ISSUE #9: Frontend Site security
- [x] #10
- [x] #11
- [x] #12
- [x] #13
- [x] #14

- [ ] ALL LOGIN PAGES SHOULD REDIRECT BACK OR TO THE MAIN PAGE

### LINKED ISSUE #10: The navigation bar should have a login button
- [ ] Dependent on #2 
- [ ] Dependent on #3 
- [ ] Login button only shows if signed out
- [ ] Log out button only shows if signed in

### LINKED ISSUE #12: Add tickets should require being signed in
REQUIREMENTS:
- [ ] If not signed in, go to login page.

### LINKED ISSUE #13: Edit tickets should require being signed in
REQUIREMENTS:
- [ ] If not signed in, go to login page.

### LINKED ISSUE #14: Disabling tickets should require being signed in
REQUIREMENTS:
- [ ] If not signed in, go to login page.

### PR DESCRIPTION
**Pull request by Annie021**

### CODE REVIEW

**Review by gtaEPIC — CHANGES_REQUESTED** [review]
#23

File: src/App.js
Comment (gtaEPIC) [inline src/App.js:1]: Why are we trying to use this still for routing. index.js already has this

Comment (Annie021) [reply src/App.js:1]: I tried but app.js wasn't working

File: src/components/AddTicket.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ title, description }),
+      });
+
+      // Check if the request was successful (status code 2xx)
+      if (response.ok) {
+        console.log('New ticket added successfully!');
+        // Optionally, you can redirect or update the UI after a successful request
```
Comment (gtaEPIC) [inline src/components/AddTicket.js]: sus chatGPT usage ._.

**Review by gtaEPIC — APPROVED** [review]

**Pull request merged**